README.md

```diff
@@ -32,8 +32,8 @@ This C# SDK is automatically generated by the [Swagger Codegen](https://github.c
 - SDK version: 2.3.1
 - Build package: io.swagger.codegen.v3.generators.dotnet.CSharpClientCodegen
```

Comment:
Please make the H1 and intro paragraph consistent with the example Node README: https://github.com/fingerprintjs/fingerprintjs-pro-server-api-node-sdk

Reply:
Done!